lib/api/repository/profile_repository.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:injectable/injectable.dart';
+import 'package:survey_flutter_ic/api/service/profile_service.dart';
+import 'package:survey_flutter_ic/model/profile_model.dart';
+
+import '../exception/network_exceptions.dart';
+
+abstract class ProfileRepository {
```

Comment:
Should it be `UserRepository` 🙏

Reply:
@Wadeewee Since we don't have other endpoints for the user so I decided to name it `Profile`, it would make sense to `User` instead for common cases, let me update then 😄